apps/publish/enqueue/enqueue_service.py

```diff
@@ -187,19 +180,17 @@ def publish(self, doc, target_media_type=None):
         6. Publish the content to content api.
 
         :param dict doc: document to publish
-        :param str target_media_type: dictate if the doc being queued is a Takes Package or an Individual Article.
-                Valid values are - Wire, Digital. If Digital then the doc being queued is a Takes Package and if Wire
-                then the doc being queues is an Individual Article.
+        :param str target_media_type: Valid values are - Wire, Digital.
         :return bool: if content is queued then True else False
         :raises PublishQueueError.item_not_queued_error:
                 If the nothing is queued.
         """
         # Step 1
         subscribers, subscriber_codes, associations = self.get_subscribers(doc, target_media_type)
 
-        # Step 2
-        if target_media_type == SUBSCRIBER_TYPES.WIRE:
-            self._update_headline_sequence(doc)
+        # # Step 2
```

Comment:
Yes, it looks few commented lines escaped in this file, I'll clean them up. Thanks.